File: sep-product-service/src/main/java/com/kurtcan/sepproductservice/payment/Payment.java
package com.kurtcan.sepproductservice.payment;

import lombok.*;
import lombok.experimental.SuperBuilder;

import java.math.BigDecimal;
import java.util.UUID;

@Data
@ToString
@SuperBuilder
@NoArgsConstructor
@AllArgsConstructor
public class Payment {
    private UUID userId;
    private UUID productId;
    private int amount;
    private BigDecimal price;
}

File: sep-search-service/src/main/kotlin/com/kurtcan/sepsearchservice/payment/Payment.kt
package com.kurtcan.sepsearchservice.payment

import com.fasterxml.jackson.annotation.JsonCreator
import com.fasterxml.jackson.annotation.JsonProperty
import com.kurtcan.sepsearchservice.base.BaseEntity
import java.math.BigDecimal
import java.util.*

data class Payment @JsonCreator constructor(
    @JsonProperty("userId") val userId: UUID,
    @JsonProperty("productId") val productId: UUID,
    @JsonProperty("amount") val amount: Int,
    @JsonProperty("price") val price: BigDecimal
) : BaseEntity()